src/ApiService/ApiService/QueueNodeHearbeat.cs

```diff
@@ -25,7 +25,7 @@ public QueueNodeHearbeat(ILoggerFactory loggerFactory, IStorageProvider storageP
     [Function("QueueNodeHearbeat")]
     public async Task Run([QueueTrigger("myqueue-items", Connection = "AzureWebJobsStorage")] string msg)
     {
-        var hb = JsonSerializer.Deserialize<NodeHeartbeatEntry>(msg, EntityConverter.GetJsonSerializerOptions());
+        var hb = JsonSerializer.Deserialize<NodeHeartbeatEntry>(msg, EntityConverter.GetJsonSerializerOptions()).EnsureNotNull($"wrong data {msg}");
```

Comment:
Use this ? 

https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-coalescing-operator

Reply:
no the same thing. I want to convert nullable to a non nullable and throw if it is not the case. 
with the operator the code will look like 
``` C#
var hb = JsonSerializer.Deserialize<NodeHeartbeatEntry>(msg, EntityConverter.GetJsonSerializerOptions()) ?? throw new NullReferenceException($"wrong data {msg}");
```
The extension method is just a shortcut